client/containers/Pub/PubDocument/Review/ReviewerDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Button, Classes, Dialog, InputGroup } from '@blueprintjs/core';
+
+require('./reviewerDialog.scss');
+
+type Props = {
+	isOpen: boolean;
+	pubData: {
+		title: string;
+	};
+	onClose: (...args: any[]) => any;
+	createReviewDoc: () => any;
```

Comment:
Conventionally, event handlers passed into a component start with `on`. So this would be `onCreateReviewDoc`.